services/account_type/master_payer.yaml

```diff
@@ -97,19 +93,22 @@ Resources:
           - Sid: ReservedCapacityServices
             Effect: Allow
             Action:
-              - 'ec2:Describe*'
-              - 'ec2:Get*'
-              - 'ec2:Get*'
-              - 'rds:Describe*'
+              - 'cloudfront:Get*'
+              - 'cloudfront:List*'
               - 'dynamodb:Describe*'
               - 'dynamodb:Get*'
               - 'dynamodb:List*'
-              - 'cloudfront:Get*'
-              - 'cloudfront:List*'
+              - 'ec2:Describe*'
+              - 'ec2:Get*'
+              - 'ec2:Get*'
               - 'elasticache:Describe*'
               - 'elasticache:List*'
               - 'elasticmapreduce:Describe*'
               - 'elasticmapreduce:List*'
+              - 'elasticsearch:Describe*'
```

Comment:
The Elasticsearch Service IAM actions are `es:*`, see https://docs.aws.amazon.com/elasticsearch-service/latest/developerguide/es-ac.html#es-ac-reference.